### PR TITLE
Default to `pointer-events: none` on Sortable helper

### PR DESCRIPTION
**Pull request by clauderic**
Because... trackpad ninjas.

https://twitter.com/Vjeux/status/840264657424146436

### CODE REVIEW

File: src/SortableContainer/index.js [hunk 1]
```diff
@@ -288,6 +288,7 @@ export default function sortableContainer(WrappedComponent, config = {withRef: f
         this.helper.style.width = `${this.width}px`;
         this.helper.style.height = `${this.height}px`;
         this.helper.style.boxSizing = 'border-box';
+	this.helper.style.pointerEvents = 'none';
```
Comment (bvaughn) [inline src/SortableContainer/index.js]: White space police! 🚓   Tab spotted!👮

Comment (clauderic) [inline src/SortableContainer/index.js]: 😂

The one time I'm lazy and decide to use the built-in github editor 🙄

Comment (clauderic) [inline src/SortableContainer/index.js]: Never will I ever again.

Comment (bvaughn) [inline src/SortableContainer/index.js]: GOTCHA! 🥇

Comment (rauchg) [inline src/SortableContainer/index.js]: Prettier-bot would not have allowed this 🤖

Comment (clauderic) [inline src/SortableContainer/index.js]: (╯°□°）╯︵ ┻━┻

Comment (bvaughn) [inline src/SortableContainer/index.js]: It may be time to step away from open source, Clauderic

**Comment by ahmedisam99** [comment]
Why!! just why??!!?
don't you think you should use a property to allow users to specify whether to disable pointer events or not? @clauderic 
It's causing a trouble with all users wanting to specify a custom cursor while dragging the list-item.

**Comment by josias-r** [comment]
@clauderic Took me a while until I finally found why the cursor is not behaving as it is in the examples. Ahmedisam99 pointed it out already, in a rude manner though, that this change breaks the cursor styles. Would be nice if this was configurable. Also it's misleading that the Examples are not up-to-date where this change has not yet been made.

**Pull request merged**